Linkbrary/javascript/register.js

```diff
@@ -0,0 +1,16 @@
+// 회원가입 시도
+function RegisterTry(event) {
```

Comment:
함수 이름은 보통 카멜케이스를 따르고, 클래스명이나 (나중에 배우실) 컴포넌트에만 대문자를 쓰는 편입니다!
```suggestion
function register(event) {
```